### ISSUE TITLE
Question about Seq2Seq training code

### ISSUE THREAD

**Issue by kyquang97** [pytorch/tutorials#501]
Hi pytorch team, i'm learning seq2seq translation with pytorch tutorial. In the training code of tutorial, it's use topk method to get topv (top values) and topi (top indices) of a tensor. Then the author assign decoder_input is equal to topi.squeeze().detach(). I think it should be topv because topi is top indices of max values in tensor. Is this a mistake ?
